src/index.js

```diff
@@ -0,0 +1,47 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './style.css';
+
+const inputField = document.querySelector('#inputField');
+const todoList = document.querySelector('#todoList');
+
+const todos = [
+  {
+    description: 'wash my hair',
+    index: 1,
+    completed: 0,
+  },
+  {
+    description: 'sleep',
+    index: 2,
+    completed: 0,
+  },
+];
```

Comment:
- [ ] when I load my local server, the to-do items do not show. It's only after adding a task in the input field that the above two tasks show plus the additionally added todo. For this milestone, you are only required to display todos that you have placed in the array above and render them dynamically using a loop as I have seen you do below but not implementing the logic for adding a todo, because it seems to bring that issue.

Reply:
- [ ] Fixed